includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -837,10 +879,7 @@ private function process_css_list( CSSList $css_list, $options ) {
 					$css_list->remove( $css_item );
 				}
 			} elseif ( $css_item instanceof Import ) {
-				$validation_errors[] = array(
-					'code'    => 'illegal_css_import_rule',
-					'message' => __( 'CSS @import is currently disallowed.', 'amp' ),
-				);
+				$this->parse_import_stylesheet( $css_item );
```

Comment:
@westonruter Still WIP but started adding the support for @imported stylesheets, any thought on the approach?

Reply:
Yes, your approach is great. I went ahead and iterated on it because of the changes introduced by the refactored error validation handling.